plugins/src/opengrok/auth/plugin/LdapAttrPlugin.java

```diff
@@ -107,8 +109,12 @@ public void fillSession(HttpServletRequest req, User user) {
         if (attributeValues != null) {
             sessionAllowed = attributeValues.stream().anyMatch((t) -> whitelist.contains(t));
         } else {
-            if ((records = getLdapProvider().lookupLdapContent(user, new String[]{ldapAttr})) == null) {
-                return;
+            try {
+                if ((records = getLdapProvider().lookupLdapContent(user, new String[]{ldapAttr})) == null) {
```

Comment:
can it return null now with your changes?

Reply:
which changes do you mean ?

Reply:
specifically the changes that you throw an exception instead of returning null

Reply:
I think it can return null if `processResult()` returns `null`. The `SearchResult` class is not really specific in when it can return null; it contains some curious comments such as:
```java

    /**
     * Sets the attributes of this search result to <code>attrs</code>.
     * @param attrs The non-null attributes to use. Can be empty.
     * @see #getAttributes
     */
    public void setAttributes(Attributes attrs) {
        this.attrs = attrs;
        // ??? check for null?
    }
```

so my guess is that `null` still might be returned.

Reply:
ok